shell/components/fleet/FleetIntro.vue

```diff
@@ -1,31 +1,25 @@
 <script>
-import { AS, _YAML } from '@shell/config/query-params';
 import { FLEET } from '@shell/config/types';
+import { NAME } from '@shell/config/product/fleet';
 
 export default {
 
   name: 'FleetIntro',
 
   data() {
-    const params = { ...this.$route.params };
-
-    const formRoute = { name: `${ this.$route.name }-create`, params };
-
-    const hasEditComponent = this.$store.getters['type-map/hasCustomEdit'](this.resource);
-
-    const yamlRoute = {
-      name:  `${ this.$route.name }-create`,
-      params,
-      query: { [AS]: _YAML },
+    const gitRepoRoute = {
+      name:   'c-cluster-product-resource-create',
+      params: {
+        product:  NAME,
+        resource: FLEET.GIT_REPO
+      },
```

Comment:
The link could be hardcoded. It was only ever used for Adding Repositories anyway.